test/image/mocks/geo_lakes-and-rivers.json

```diff
@@ -0,0 +1,264 @@
+{
```

Comment:
This mock currently looks like:

![newplot (42)](https://user-images.githubusercontent.com/6675409/61243597-51806980-a716-11e9-9875-84332c17e4a4.png)

Reply:
Using topojson files from before https://github.com/plotly/plotly.js/pull/3856 fixes the lakes polygon, but the scoped rivers were still off, see https://github.com/plotly/plotly.js/pull/4048/commits/7075d136b08890741f454873d99b350e4fed7045 or diff in https://github.com/plotly/plotly.js/pull/4048/commits/5385673ed988d82dbab4be399bfce6b1f63bc01a